packages/gasket-cli/lib/commands/create.js

```diff
@@ -0,0 +1,194 @@
+      installation. (e.g. --package-manager yarn)`
+    },
+    {
+      name: 'require',
+      short: 'r',
+      description: 'Require module(s) before Gasket is initialized',
+      parse: commasToArray
+    },
+    {
+      name: 'bootstrap',
+      description: '(INTERNAL) If provided, skip the bootstrap phase',
+      default: false,
+      type: 'boolean',
+      hidden: true
+    },
+    {
+      name: 'generate',
+      description: '(INTERNAL) If provided, skip the generate phase',
+      default: false,
+      type: 'boolean',
+      hidden: true
+    },
+    {
+      name: 'npm-link',
+      description: `(INTERNAL) Local packages to be linked. Can be set as
+      multiple flags (e.g. --npm-link @gasket/plugin-jest --npm-link some-test-preset)
+      comma-separated values: --npm-link=@gasket/plugin-jest,some-test-preset`,
+      parse: commasToArray,
+      hidden: true
+    },
+    {
+      name: 'preset-path',
+      description: `(INTERNAL) Paths the a local preset packages. Can be absolute
+      or relative to the current working directory.
+      comma-separated values: --preset-path=path1,path2`,
+      parse: commasToArray,
+      hidden: true
+    },
+    {
+      name: 'config',
+      description: 'JSON object that provides the values for any interactive prompts'
+    },
+    {
+      name: 'config-file',
+      description: 'Path to a JSON file that provides the values for any interactive prompts',
+      conflicts: ['config']
+    },
+    {
+      name: 'prompts',
+      description: '(INTERNAL) Disable to skip the prompts',
+      default: true,
+      type: 'boolean',
+      hidden: true
+    }
+  ]
+};
+
+/**
+ * bootstrap - Bootstrap the application
+ * @param {CreateContext} context - Create context
+ */
+async function bootstrap(context) {
+  await loadPreset(context);
+  cliVersion(context);
+  applyPresetConfig(context);
+  await globalPrompts(context);
+  await mkDir(context);
+  await setupPkg(context);
+  await writePkg(context);
+  await installModules(context);
+  await linkModules(context);
+}
+
+/**
+ * generate - Generate the application
+ * @param {CreateContext} context - Create context
+ */
+async function generate(context) {
+  await promptHooks(context);
+  await createHooks(context);
+  await generateFiles(context);
+  await writeGasketConfig(context);
+  await writePkg.update(context);
+  await installModules.update(context);
+  await linkModules.update(context); // relink any that were messed up by re-install
+  await postCreateHooks(context);
+}
+
+/**
+ * createCommand action
+ * @param {string} appname Required cmd arg - name of the app to create
+ * @param {object} options cmd options
+ * @param {Command} command - the command instance
+ * @returns {Promise<void>} void
+ */
+createCommand.action = async function run(appname, options, command) {
+  const argv = [appname];
+  const { bootstrap: noBootstrap, generate: noGenerate } = options;
```

Comment:
See previous comment, but effectively this would be:
```suggestion
  const { bootstrap = true, generate = true } = options;
```